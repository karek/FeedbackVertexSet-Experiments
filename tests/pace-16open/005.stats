62,159,12
