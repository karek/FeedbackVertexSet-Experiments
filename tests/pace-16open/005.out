19
2 1 30 25 10 22 58 29 12 11 56 23 8 35 26 37 15 6 55 
